src/app/pages/swap/components/swap-selected-asset-from.tsx

```diff
@@ -0,0 +1,95 @@
+import { FiChevronDown } from 'react-icons/fi';
+
+import { Box, Stack, Text } from '@stacks/ui';
+import { useField, useFormikContext } from 'formik';
+
+import { isUndefined } from '@shared/utils';
+
+import { formatMoneyWithoutSymbol } from '@app/common/money/format-money';
+import { SelectedAssetField } from '@app/components/forms/selected-asset-field';
+import { Flag } from '@app/components/layout/flag';
+import { SpaceBetween } from '@app/components/layout/space-between';
+
+import { useAmountAsFiat } from '../hooks/use-amount-as-fiat';
+import { SwapFormValues } from '../hooks/use-swap';
+import { tempExchangeRate } from '../hooks/use-swappable-assets';
+import { SwapAmountField } from './swap-amount-field';
+import { SwapSelectedAssetLayout } from './swap-selected-asset.layout';
+
```

Comment:
How about moving the text into a config file?

Reply:
I don't totally understand what you mean here? Just a separate file co-located here?

Reply:
Yeah just to take it away from this logic. Again it's not really a big deal, just mentioning as I see it.

All our text is embedded into the code so it would be nice to separate it out